templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="#">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>

```

Comment:
The `href` attributes currently contain `#`, which means the links do not navigate to any specific page. Consider replacing `#` with actual URLs or Django URL template tags to make the navigation functional.